.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
FYI: the .NET Dapr Client SDK has this template as well, but it doesn't actually show up in the list of issue types (and GH complains about an invalid name). I wasn't able to find the cause of the error, so I left it out of the .NET Dapr Pluggable Components SDK.

Reply:
The issue happens because there are two config files in ISSUES_TEMPLATE in the dotnet SDK with the same `name` value:

https://github.com/dapr/dotnet-sdk/blob/blob/22ac4c51458c1ac83c72fbe15fc65289f70f8cc2/.github/ISSUE_TEMPLATE/discussion.md?plain=1#L2


https://github.com/dapr/dotnet-sdk/blob/22ac4c51458c1ac83c72fbe15fc65289f70f8cc2/.github/ISSUE_TEMPLATE/feature_request.md?plain=1#L2

Changing the name on `discussion.md` should fix the problem.

Reply:
https://github.com/dapr/dotnet-sdk/pull/1026